Docker/build/WDQS/runUpdate.sh

```diff
@@ -4,6 +4,6 @@
 cd /wdqs || exit
 
 # TODO env vars for entity namespaces, scheme and other settings
-/wait-for-it.sh "$WIKIBASE_HOST:80" -t 300 -- \
+/wait-for-it.sh "$WIKIBASE_HOST:$WIKIBASE_PORT" -t 300 -- \
 /wait-for-it.sh "$WDQS_HOST:$WDQS_PORT" -t 300 -- \
-./runUpdate.sh -h http://"$WDQS_HOST":"$WDQS_PORT" -- --wikibaseUrl "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --conceptUri "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --entityNamespaces "$WDQS_ENTITY_NAMESPACES"
+./runUpdate.sh -h http://"$WDQS_HOST":"$WDQS_PORT" -- --wikibaseUrl "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST":"$WIKIBASE_PORT" --conceptUri "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --entityNamespaces "$WDQS_ENTITY_NAMESPACES"
```

Comment:
I've added the $WIKIBASE_PORT to the wikibaseUrl in case we use something else than 80/443

Reply:
Did you get this working? I think you need to add this to the Dockerfile too